Description: msfconsole fails to start
 msfconsole fails to start when the package ruby-rbnacl is installed.
 Drop an upstream commit to fix this.
Origin: https://github.com/rapid7/metasploit-framework/commit/3d76c365118584afd7b631cbf3ae8c370a379d0b
Bug-Kali: https://bugs.kali.org/view.php?id=4467
Last-Update: 2018-01-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/metasploit-framework.gemspec
+++ b/metasploit-framework.gemspec
@@ -126,6 +126,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'mqtt'
   spec.add_runtime_dependency 'net-ssh'
   spec.add_runtime_dependency 'rbnacl', ['< 5.0.0']
+  spec.add_runtime_dependency 'rbnacl-libsodium'
   spec.add_runtime_dependency 'bcrypt_pbkdf'
   spec.add_runtime_dependency 'ruby_smb'
 
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -39,6 +39,7 @@ PATH
       railties
       rb-readline
       rbnacl (< 5.0.0)
+      rbnacl-libsodium
       recog
       redcarpet
       rex-arch
@@ -251,6 +252,8 @@ GEM
     rb-readline (0.5.5)
     rbnacl (4.0.2)
       ffi
+    rbnacl-libsodium (1.0.16)
+      rbnacl (>= 3.0.1)
     recog (2.1.17)
       nokogiri
     redcarpet (3.4.0)
